index.html

```diff
@@ -1,11 +1,15 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Welcome to GitHub Team Cooperation</title>
 </head>
+
 <body>
-    
+    <h1>This is my welcome page</h1>
+    <p>feihua</p>
```

Comment:
Could you remove the 'feihua' paragraph and make the 'This is my welcome page' red?

Reply:
Ok